docs/data/introduction/licensing/licensing.md

```diff
@@ -120,7 +120,7 @@ The number of seats purchased on your license must correspond to the number of c
   'AppY' has five front-end developers, and 'AppZ' has three; additionally, there are two front-end developers on the company's UI development team.
   Company 'B' must purchase ten seats.
 
-This is [the relevant clause in the EULA.](https://mui.com/legal/mui-x-eula/#required-quantity-of-licenses)
+This is [the relevant clause in the EULA.](https://mui.com/legal/mui-x-eula/#2-5-required-quantity-of-licenses)
```

Comment:
This change and the change in L299 do not seem to work as expected. 🤷 
The current link seems to be redirected to the correct section sometimes on some browsers... 🙈 🤷 
@oliviertassinari is there some logic on store handling this?
If there is, can we avoid it to avoid the unclear behavior? 🙈

Reply:
🤔 The original link is working as expected, doesn't it?

I believe that an anchor link without the "clause" number is more proof against update errors.

Reply:
Given the merged fix, yes, they do work fine and I'll ditch these updates. 😉

Reply:
```suggestion
This is [the relevant clause in the EULA.](https://mui.com/legal/mui-x-eula/#required-quantity-of-licenses)
```